docs/Elasticsearch-batchsink.md

```diff
@@ -27,6 +27,9 @@ exist, it will be created. (Macro-enabled)
 **es.idField:** The field that will determine the id for the document; it should match a fieldname
 in the Structured Record of the input. (Macro-enabled)
 
+**additionalProperties:** Additional properties to use with the es-hadoop client when writing the data, 
```

Comment:
Can we change it to "Additional Properties"

Reply:
Also, can we add a KV with K drop-down so that it becomes easier for user ?

Reply:
@nitinmotgi do you mean instead of a JSON text box, we should use a KV dropdown component? Can you point me to any doc I can refer to add it?

Reply:
@nitinmotgi please ignore the above. I got the reference from other plugins. Let me make the changes.